app/templates/menu/menu.hbs

```diff
@@ -1,2 +1,2 @@
-<div class="menu">
-</div>
\ No newline at end of file
+<div class="menu {{platform}}">
```

Comment:
Please pass it in app-view, here: https://github.com/keeweb/keeweb/blob/develop/app/scripts/views/app-view.js#L104
We don't need a helper, to use it only one time.